static/app/actionCreators/sudoModal.tsx

```diff
@@ -1,16 +1,22 @@
 import ModalStore from 'sentry/stores/modalStore';
 
 type OpenSudoModalOptions = {
+  closeButton?: boolean;
+  closeEvents?: 'all' | 'none' | 'backdrop-click' | 'escape-key';
```

Comment:
I'd recommend either reusing the modal options from `globalModal` here; https://github.com/getsentry/sentry/blob/master/static/app/components/globalModal/index.tsx#L20

```suggestion
  closeEvents?: ModalOptions.closeEvents
```

A possibly better way to handle it could be to use the whole `ModalOptions`
```
type OpenSudoModalOptions = ModalOptions & {
   // only the unique options for the OpenSudo modal
}
```

Reply:
:O